### PR TITLE
OP-1141 - Demo credential fix

### PR DESCRIPTION
**Pull request by mizzioisf**
Hopefully last PR for https://openhospital.atlassian.net/browse/OP-1141

### CODE REVIEW

File: oh.bat [hunk 1]
```diff
@@ -329,7 +329,6 @@ set CLASSPATH=%CLASSPATH%;%OH_PATH%\%OH_DIR%\bundle
 set CLASSPATH=%CLASSPATH%;%OH_PATH%\%OH_DIR%\rpt_base
 set CLASSPATH=%CLASSPATH%;%OH_PATH%\%OH_DIR%\rpt_extra
 set CLASSPATH=%CLASSPATH%;%OH_PATH%\%OH_DIR%\rpt_stat
-set CLASSPATH=%CLASSPATH%;%OH_PATH%\%OH_DIR%\rsc\images
```
Comment (mwithi) [inline oh.bat:332]: this was added recently on develop

File: oh.ps1 [hunk 2]
```diff
@@ -629,7 +629,6 @@ function java_lib_setup {
 	$script:OH_CLASSPATH="$OH_CLASSPATH;$OH_PATH\$OH_DIR\rpt_base\"
 	$script:OH_CLASSPATH="$OH_CLASSPATH;$OH_PATH\$OH_DIR\rpt_extra\"
 	$script:OH_CLASSPATH="$OH_CLASSPATH;$OH_PATH\$OH_DIR\rpt_stat\"
-	$script:OH_CLASSPATH="$OH_CLASSPATH;$OH_PATH\$OH_DIR\rsc\images\"
```
Comment (mwithi) [inline oh.ps1:632]: this was added recently on develop

File: oh.sh [hunk 3]
```diff
@@ -577,7 +577,6 @@ function java_lib_setup {
 	OH_CLASSPATH=$OH_CLASSPATH:"$OH_PATH"/$OH_DIR/rpt_base
 	OH_CLASSPATH=$OH_CLASSPATH:"$OH_PATH"/$OH_DIR/rpt_extra
 	OH_CLASSPATH=$OH_CLASSPATH:"$OH_PATH"/$OH_DIR/rpt_stat
-	OH_CLASSPATH=$OH_CLASSPATH:"$OH_PATH"/$OH_DIR/rsc/images
```
Comment (mwithi) [inline oh.sh:580]: this was added recently on develop

File: oh.ps1 [hunk 4]
```diff
@@ -1075,7 +1074,7 @@ function write_config_files {
 		copy_config_file $CRED_SETTINGS;
 	}
 	if ( $DEMO_DATA -eq "on" ) {
-		copy_config_file $DEMO_CRED_SETTINGS;
+		(Get-Content "$OH_PATH/$OH_DIR/rsc/$DEMO_CRED_SETTINGS.dist") | Set-Content "$OH_PATH/$OH_DIR/rsc/$CRED_SETTINGS"
 	}
 }
```
Comment (mwithi) [inline oh.ps1:1079]: it could be possible that it is demo_data and not portable?

Comment (mizzioisf) [reply oh.ps1:1079]: No, not possibile.

**Comment by mizzioisf** [comment]
Everything shoud be ok now.

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**